starter-code/public/javascripts/financial-data.js

```diff
@@ -0,0 +1,85 @@
+function main () {
+  const coinDeskApi = axios.create({
+    baseURL: 'http://api.coindesk.com/v1/bpi/historical/close.json'
+  });
+
+  function getData (id) {
+    return coinDeskApi.get(id)
+      .then(response => response.data.bpi);
+  }
+
+  function filterData (data) {
+    console.log('filter data console log');
+    // NOT WORKING -- is a work in progress
+    const startValue = document.getElementById('start').value;
+    const endValue = document.getElementById('end').value;
+    const filteredArr = [];
+    data.forEach(function (data) {
```

Comment:
You have an error, data is undefined. Why? Because you call the function without passing any arguments (line 74).